model/docs/example_with_datasets.qmd

```diff
@@ -370,6 +426,7 @@ out2 = hosp_model.plot_posterior(
     var="all_latent_infections", ylab="Latent Infections"
 )
 ```
+
 and the distribution of latent infections
```

Comment:
Edit this text and the text before the previous figure to differentiate them something like "We can look at individual draws from the posterior distribution of latent infections" and then "We can also look at credible intervals for the posterior distribution of latent infections"